submit.go

```diff
@@ -120,11 +120,13 @@ type submitResponse struct {
 	ReportURL string `json:"report_url,omitempty"`
 }
 
-// regex to catch and substitute ambiguous issue references with explicit ones to the actual repo they are in
-var ambiguousIssueRegex = regexp.MustCompile(`(^|[([{\s])(#\d+)([^\w]|$)`)
+// regex to match and substitute ambiguous issue references in the rageshake body text
+// matches a hash followed by digits optionally surrounded by whitespace or some punctuation
+// also matches if the input starts with digits where the hash becomes optional
```

Comment:
I'm a bit surprised that you really want to match things that start with digits... is this is a common pattern?

in any case: imho there comes a point where you should just use a separate `if` condition, rather than an increasingly complex regex...